src/app/business/delfin/storages/storages.component.ts

```diff
@@ -93,6 +95,12 @@ export class StoragesComponent implements OnInit {
 
     };
 
+    perfMetricsConfigFormLabel = {
+        "perf_collection": "Enable Performance collection?",
```

Comment:
Can you please check and confirm if putting a '?' is fine from Ux perspective? As this is Bool (Yes/No), it's quite obvious

Reply:
Wil fix this in the subsequent PRs as this will be updated in the next PR